tests/integration/cos_substrate.py

```diff
@@ -18,11 +18,7 @@ class COSSubstrate(Protocol):
     """Interface for managing a COS substrate."""
 
     def create_substrate(self) -> str:
-        """Create a COS substrate.
```

Comment:
now you're into @mateoflorido 's business.

Reply:
The linter was quite angry .... I hope I didn't step on anyones toes :)

Reply:
we can't have angry linters.  Surely @mateoflorido appreciates it